packages/client/src/Play-Game/CreateGamePage.js

```diff
@@ -48,7 +48,7 @@ const CreateGamePage = () => (
       mutation={gql`
         mutation($createMatch: NewMatchInput!) {
           createMatch(input: $createMatch) {
-            match_id
+            id
```

Comment:
This is incorrect and will break create game functionality.

Reply:
fixed.